DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: rhino
 Title: A Framework for Enterprise Shiny Applications
-Version: 1.7.0.9000
+Version: 1.7.0.9001
```

Comment:
This should be 1.6.0.9001

Reply:
This means `dev_next` should be `1.6.0.9000`